openhtf/io/proto/assembly.proto

```diff
@@ -0,0 +1,103 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
I was, but on second thought, I think it can remain internal. I'll just wrap the `_TestRunFromTestRecord` with assembly info, likely from the 'metadata' dict.